### PR TITLE
txthl class removed

### LINKED ISSUE #8755: Text highlighting
Am comparing https://wet-boew.github.io/wet-boew-styleguide/design/textediting-en.html#marked and https://wet-boew.github.io/v4.0-ci/docs/ref/texthighlight/texthighlight-en.html. The latter uses a brighter yellow. Shouldn't they be the same?

### PR DESCRIPTION
**Pull request by namjohn920**
txthl class removed and <mark> tag changed to have background as #ffff00. 

fixes #8755

### CODE REVIEW

**Review by duboisp — CHANGES_REQUESTED** [review]
Can you include your Non-text contrast test in the plugin documentation? Also, please share your result in a comments in the results. 

Add a new section in the documentation E & F.

File: site/pages/docs/ref/texthighlight/texthighlight-fr.hbs [hunk 1]
```diff
@@ -26,6 +26,17 @@
...
 
+{{!-- Section for non-text contrast test.
+The ratio should be greater than 3:1.
+The background color used for highlight is #ffff00 and the font color is #000000, resulting in ratio of 19.55:1.
+--}}
+<section>
+	<h2>Non-text contrast</h2>
+	<p>According to <a href="https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html">Understanding Success Criterion 1.4.11: Non-text Contrast</a>, the contrast ratio is required to be at least 3:1.</p>
+	<p>The background color used for highlight is #ffff00 and the font color is #000000, resulting in ratio of 19.55:1</p>
+	<img src="../../img/contrast_ratio.png" alt="contrast_ratio">
```
Comment (GormFrank) [inline site/pages/docs/ref/texthighlight/texthighlight-fr.hbs]: You do not need to put an image here.

File: site/pages/docs/ref/texthighlight/texthighlight-en.hbs [hunk 2]
```diff
@@ -26,6 +26,17 @@
...
 
+{{!-- Section for non-text contrast test.
+The ratio should be greater than 3:1.
+The background color used for highlight is #ffff00 and the font color is #000000, resulting in ratio of 19.55:1.
+--}}
+<section>
+	<h2>Non-text contrast</h2>
+	<p>According to <a href="https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html">Understanding Success Criterion 1.4.11: Non-text Contrast</a>, the contrast ratio is required to be at least 3:1.</p>
+	<p>The background color used for highlight is #ffff00 and the font color is #000000, resulting in ratio of 19.55:1</p>
+	<img src="../../img/contrast_ratio.png" alt="contrast_ratio">
```
Comment (GormFrank) [inline site/pages/docs/ref/texthighlight/texthighlight-en.hbs]: You do not need to put an image here. You can get rid of the image file.  Worst case you could put a link to your contrast checker if you really want to. Also, just as an FYI, `alt` attributes in `img` tags should be written in human readable language.

Apart from that, good to go.

**Review by GormFrank — CHANGES_REQUESTED** [review]

**Review by GormFrank — APPROVED** [review]

File: src/plugins/texthighlight/_base.scss [hunk 3]
```diff
@@ -2,10 +2,8 @@
  * Web Experience Toolkit (WET) / Boîte à outils de l'expérience Web (BOEW)
  * wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
  */
-.wb-txthl {
-	.txthl {
-		background-color: $txthl-bg-color;
-		color: $txthl-color;
-		font-weight: $txthl-font-weight;
-	}
+mark {
```
Comment (duboisp) [inline src/plugins/texthighlight/_base.scss]: This code would need to be moved into the bootstrap override CSS.

**Review by duboisp — CHANGES_REQUESTED** [review]

**Comment by GormFrank** [comment]
Pre-approved at technical review as long as the recent changes are made.

File: src/plugins/texthighlight/_base.scss (hunk 3 above)
Comment (GormFrank) [reply src/plugins/texthighlight/_base.scss]: @duboisp Resolved!

**Review by duboisp — APPROVED** [review]

**Pull request merged**